deps/source_package/debian/control

```diff
@@ -0,0 +1,64 @@
+Source: edubuntu-meta
+Section: metapackages
+Priority: optional
+Maintainer: Deepanshu Gajbhiye <gajbhiye.deepanshu@gmail.com>
+Standards-Version: 4.1.4
+Build-Depends: debhelper (>= 11), germinate (>= 1.18)
+
+Package: debian-desktop
+Architecture: amd64 armhf arm64 i386 powerpc ppc64el
+Depends: ${germinate:Depends}, ${misc:Depends}
+Recommends: ${germinate:Recommends}
+Description: educational desktop for Debian
+ This package depends on the complete set of Ed-supported packages. It
+ provides an educational desktop on top of Debian. It includes:
+  * Edubuntu artwork and documentation
+  * Preschool application bundle
+  * Primary level application bundle
+  * Secondary level application bundle
+  * Tertiary level application bundle
+  * Miscellaneous applications useful in educational settings
+ .
+ It is safe to remove this package if some of the desktop system packages are
+ not desired.
+ 
+Package: debian-fonts
+Architecture: any
+Depends: ${germinate:Depends}, ${misc:Depends}
+Recommends: ${germinate:Recommends}
+Description: Additional fonts included on a default debian system
+ This metapackage will install a selection of packages that contain
+ fonts that may be useful for education or desktop publishing.
+ 
+Package: debian-edu-preschool
+Architecture: any
+Depends: calibre, chemtool, fritzing, inkscape, kalzium, kmplot, ktouch, kturtle, lightspeed, marble, melting, pencil2d, rocs, step, yorick, ${germinate:Depends}, ${misc:Depends}
```

Comment:
Yes. But there is a little problem.  One cannot install recommended and dependent packages with `dpkg`. So we have to use `apt install -f` to fix dependencies.

We install recommended packages with flag `--install-recommends`. But combining the 2 will not work. I am trying to solve this one.

Reply:
No. If you publish your deb file and you used `apt install debian-edu-*` it will install recommended package too. - `apt-get` does so. Why don't you setup PPA and upload these deb files and try to install it?

Reply:
I was thinking of this. But I saw this at https://wiki.debian.org/Packaging/Intro - 
```
QUESTION: is there anything like PPA in Ubuntu?
ANSWER: Debian does not run a PPA service like Ubuntu, but all the tools to make apt repositories yourself are there, it's just a lot to configure by oneself.
```
But still we can do this. Since there is no other way of installing recommended packages other than  `apt`

Reply:
> If you publish your deb file and you used apt install debian-edu-* it will install recommended package too.

The command `apt install ...` works with deb files too. No need for a PPA or something similar.

> However debian has mentors.debian.net I am uploading my packages there

First install them internally (what you have done so far). Uploading them to debian should be the next step, because it is more complex and requires more responsibility. You don't just upload them to debian but you promise to maintain them for a long time, even after the GSoC project is finished, and you take responsibility for them.

As I have told you previously, it is better to find some EDU related project on debian, and submit them to this project (or through this project, I don't know how it works), rather than submitting them directly to debian.

Reply:
There is a problem with `apt install debian-edu-*.deb`, that we don't give the user a choice about which packages to install.
See: https://github.com/docker-scripts/dev--LTSP/issues/59